components/Text/Display.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+type DisplayProps = {};
+
+const Display: FC<DisplayProps> = ({ ...props }) => {
```

Comment:
What's this component for?

Reply:
This corresponds with a variant of text styles in the Figma designs. I don't think we use it anywhere yet, but I created the component as a reminder. I'll add a note in the checklist.

Reply:
ok not sure what sort of text style this is but sounds quite abstract to me, could be good to review some of the naming as would be nice to make things as clear as possible, appreciate it's never an easy task but I think it's quite an important factor of a component library.